playbooks/hc-ansible-deployment/nx3_gluster_inventory.yml

```diff
@@ -125,7 +125,6 @@ hc_nodes:
       - 2049/tcp
       - 54321/tcp
       - 5900/tcp
```

Comment:
same here, 5900 still exists.

Reply:
So we just don't need 6900/tcp port
Because that is required by Hosted Engine